services/ui-src/package.json

```diff
@@ -75,6 +75,7 @@
     "react-redux": "^7.2.2",
     "react-router-bootstrap": "^0.25.0",
     "react-router-dom": "^5.1.2",
+    "react-scripts": "^5.0.0",
```

Comment:
switched from webpack to react-scripts

Reply:
Does that allow us to remove the webpack dependencies in this file?

Reply:
I believe it does! I decided to leave it for now so that this upgrade could go through, then have a follow-up to remove everything for webpack. I hope it's as easy as deleting the dependencies and files, though I dare not assume webpack would let me get away so easily